src/test/operate.test.js

```diff
@@ -0,0 +1,24 @@
+import operate from '../logic/operate';
+
+describe('operate', () => {
+  it('should return the sum of numbers', () => {
+    const newData = operate(5, 20, '+');
+    expect(newData).toEqual('25');
+  });
+  it('should return the rest of numbers', () => {
+    const newData = operate(5, 20, '-');
+    expect(newData).toEqual('-15');
+  });
+  it('should return the multiplication of numbers', () => {
+    const newData = operate(5, 20, 'x');
+    expect(newData).toEqual('100');
+  });
+  it('should return the multiplication of numbers', () => {
```

Comment:
I think it should be `  it('should return the division of numbers', () => {`

Reply:
thanks